bullet.cpp

```diff
@@ -0,0 +1,64 @@
+#include "bullet.h"
+#include "glm/gtc/matrix_transform.hpp"
+#include "glm/gtc/type_ptr.hpp"
+#include <glm/ext.hpp>
+#include <iostream>
+#include <cmath>
+
+#define PI 3.14159265
+
+Bullet::Bullet(const vector<GLuint> textures, glm::vec3 pos, glm::vec3 direction, const float scale):
+	textures(textures), scale(scale)
```

Comment:
Could you reverse the order of initialization here? The compiler is throwing warnings about it.